Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.h

```diff
@@ -105,12 +114,15 @@ class GDKTESTGYMS_API ABenchmarkGymGameModeBase : public AGameModeBase
 	void TickClientFPSCheck(float DeltaSeconds);
 	void TickUXMetricCheck(float DeltaSeconds);
 	void TickActorCountCheck(float DeltaSeconds);
+	void TickActorMigration(float DeltaSeconds);
 
+	int GetMovementAuthActors() const;
 	void SetTotalNPCs(int32 Value);
 
 	double GetClientRTT() const { return AveragedClientRTTSeconds; }
 	double GetClientUpdateTimeDelta() const { return AveragedClientUpdateTimeDeltaSeconds; }
 	double GetPlayersConnected() const { return static_cast<double>(ActivePlayers); }
+	double GetTotalMigrations() const;
 	double GetFPSValid() const { return !bHasFpsFailed ? 1.0 : 0.0; }
```

Comment:
Are we going to add an GetMigrationsValid() method?

Reply:
Yes, only keep a GetTotalMigrationValid() is better.